src/test/kotlin/no/nav/familie/dokument/storage/StonadControllerTest.kt

```diff
@@ -41,28 +39,9 @@ internal class StonadControllerTest {
     @Test
     internal fun `skal feile ved mellomlagring dersom søknaden ikke er gyldig json`() {
         val ugyldigJson = "Jeg gikk en tur på stien"
-        assertThrows<IllegalStateException> { stonadController.mellomlagreSøknad(StonadController.StønadParameter.valueOf("overgangsstonad"), ugyldigJson) }
+        assertThrows<IllegalArgumentException> {
+            stonadController.mellomlagreSøknad(StonadController.StønadParameter.valueOf("overgangsstonad"),
+                                               ugyldigJson)
+        }
     }
-
-    @Test
-    internal fun `skal feile ved mellomlagring dersom s3 kaster exception`() {
```

Comment:
I think these tests still can be useful, making sure the API haven't changed, so consumers does not need to change anything

Reply:
Well, I see now, these were changed to StonadControllerIntegrasionTest instead?